mne/viz/tests/test_raw.py

```diff
@@ -440,33 +423,32 @@ def test_plot_raw_traces(raw):
     with pytest.warns(RuntimeWarning, match='outside data range'):
         raw.set_annotations(annot)
     fig = plot_raw(raw, events=events, event_color={-1: 'r', 998: 'b'})
-    plt.close('all')
+    browse_backend._close_all()
     for group_by, order in zip(['position', 'selection'],
                                [np.arange(len(raw.ch_names))[::-3],
                                 [1, 2, 4, 6]]):
         fig = raw.plot(group_by=group_by, order=order)
         x = fig.get_axes()[0].lines[1].get_xdata()[10]
         y = fig.get_axes()[0].lines[1].get_ydata()[10]
-        _fake_click(fig, data_ax, [x, y], xform='data')  # mark bad
```

Comment:
I think this line is the reason, why the refactored test fails. Is it intentional that the _fake_click is transformed to data-coordinates of data_ax instead of fig.ax_main from the fig instantiated three lines above?

Reply:
this was probably a mistake / cruft.